docs/FAQ.md

```diff
@@ -109,6 +109,17 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### What do I do with divergent changes?
+
+A divergent change (denoted by `??` after the change ID in the log) represents
+a change that has two or more visible commits associated with it. Most commonly,
```

Comment:
I would mention that these commits need to be referred to the commit id.

> To refer to such commits, you must use their [commit id](link to glossary). Most commonly ... abandon the unneeded commits (using their commit ids).

Reply:
Considering https://github.com/martinvonz/jj/discussions/2485, we could also say something like

> If you would like to keep both versions of the revision, you can `jj duplicate` one of them before the `jj abandon`.

Reply:
Done.